### PR TITLE
[Enhancement] Make error msg more clearly

### PR DESCRIPTION
**Pull request by TszKitLo40**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java [hunk 1]
```diff
@@ -172,7 +172,8 @@ private StorageVolume getStorageVolumeOfTable(String svName, long dbId) throws D
             } else {
                 sv = getStorageVolumeByName(BUILTIN_STORAGE_VOLUME);
                 if (sv == null) {
-                    throw new DdlException("Builtin storage volume not exists, please check the params in config");
+                    throw new DdlException("Cannot find a suitable storage volume. " +
+                            "The possible cause is that 'enable_load_volume_from_conf' is set to false during the upgrade.");
```
Comment (sduzh) [inline fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java:176]: So what should users do?

Comment (sduzh) [reply fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java:176]: You can get the value of  `Config.enable_load_volume_from_conf` here and provide a clearer prompt.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 6 / 6 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/SharedDataStorageVolumeMgr.java|6|6|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 6 / 6 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/SharedDataStorageVolumeMgr.java|6|6|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java (hunk 1 above)
Comment (TszKitLo40) [reply fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java]: @sduzh PTAL

File: fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java [hunk 2]
```diff
@@ -36,6 +39,7 @@
 import java.util.stream.Collectors;
 
 public class SharedDataStorageVolumeMgr extends StorageVolumeMgr {
+    private static final Logger LOG = LogManager.getLogger(SharedDataStorageVolumeMgr.class);
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java:42]: add a new blank line

File: fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java [hunk 3]
```diff
@@ -174,7 +178,15 @@ private StorageVolume getStorageVolumeOfTable(String svName, long dbId) throws D
             } else {
                 sv = getStorageVolumeByName(BUILTIN_STORAGE_VOLUME);
                 if (sv == null) {
-                    throw new DdlException("Builtin storage volume not exists, please check the params in config");
+                    if (Config.enable_load_volume_from_conf) {
+                        LOG.error("Failed to get builtin storage volume, svName: {}, dbId: {}, current stack trace: {}",
+                                svName, dbId, LogUtil.getCurrentStackTrace());
+                        throw new DdlException("Internal error");
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java:184]: `internal error` message is not suitable

**Review by wyb — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 7 / 7 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/SharedDataStorageVolumeMgr.java|7|7|100.00%|[]|

**Review by kevincai — APPROVED** [review]

**Review by silverbullet233 — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Pull request merged**